paasta_tools/contrib/create_paasta_playground.py

```diff
@@ -38,5 +49,29 @@ def main():
     )
 
 
+def run_local_zookeeper(user, zookeeper_port):
+    client = get_docker_client()
+    containers = client.containers()
+    zookeeper_container = next(
+        (
+            container
+            for container in containers
+            if container["Names"] == [f"/{user}-paasta-zookeeper"]
+        ),
+        None,
+    )
+    if zookeeper_container is None:
+        create_zookeeper_container(zookeeper_port, user)
+    elif zookeeper_container.get("Status") != "running":
+        client.remove_container(zookeeper_container, force=True)
+        create_zookeeper_container(zookeeper_port, user)
+
+
+def create_zookeeper_container(port, user):
+    os.system(
```

Comment:
also, we should probably be a little more consistent and use subprocess.run here rather than os.system